### PR TITLE
Translate Login and Home

### PR DESCRIPTION
**Pull request by josehernandezfintecheandomx**
## Description

Translate for Login and Home views

## Screenshots, if any
Screen Shot 2022-09-20 at 21 15 21

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] If you have multiple commits please combine them into one commit by squashing them.

- [ ] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/clients/clients-view/identities-tab/identities-tab.component.html [hunk 1]
```diff
@@ -13,16 +13,16 @@ <h3>Identities</h3>
           <td mat-cell *matCellDef="let identity">{{identity.id}}</td>
         </ng-container>
 
-        <ng-container matColumnDef="description">
```
Comment (adamsaghy) [inline src/app/clients/clients-view/identities-tab/identities-tab.component.html:16]: No need the description anymore?

File: src/app/clients/clients-view/identities-tab/identities-tab.component.html [hunk 2]
```diff
@@ -13,16 +13,16 @@ <h3>Identities</h3>
...
-          <th mat-header-cell *matHeaderCellDef> Description </th>
-          <td mat-cell *matCellDef="let identity"> {{identity.description}} </td>
-        </ng-container>
-
         <ng-container matColumnDef="type">
           <th mat-header-cell *matHeaderCellDef> type </th>
           <td mat-cell *matCellDef="let identity"> {{identity.documentType.name}}</td>
         </ng-container>
 
+        <ng-container matColumnDef="description">
```
Comment (adamsaghy) [inline src/app/clients/clients-view/identities-tab/identities-tab.component.html:21]: Please fix the "matColumnDef"

**Review by adamsaghy — COMMENTED** [review]
@josehernandezfintecheandomx Please take a quick look on my questions!

File: src/app/clients/clients-view/identities-tab/identities-tab.component.html (hunk 1 above)
Comment (josehernandezfintecheandomx) [reply src/app/clients/clients-view/identities-tab/identities-tab.component.html:16]: I don't think, but I've added too

File: src/app/core/shell/toolbar/toolbar.component.html [hunk 3]
```diff
@@ -13,31 +13,31 @@
...
       <a #selfService mat-tab-link class="tab-link" [matMenuTriggerFor]="selfServiceMenu" #selfServiceMenuTrigger="matMenuTrigger">
         <fa-icon class="mr-05" icon="users" size="lg"></fa-icon>
-        {{ 'Self Service' | translate }}
+        {{ 'labels.menus.Self Service' | translate }}
       </a>
 
       <a #configWizard mat-tab-link class="tab-link" (click)="openDialog()">
         <fa-icon class="mr-05" icon="info" size="lg"></fa-icon>
-        Configuration Wizard
+        {{ 'Configuration Wizard' | translate }}
```
[the remark this answers is not in the document]
Comment (josehernandezfintecheandomx) [reply src/app/core/shell/toolbar/toolbar.component.html]: Done!

**Pull request merged**